tools/metadata.json

```diff
@@ -1,9 +1,9 @@
 {
-  "StableReleaseTag": "v7.2.2",
+  "StableReleaseTag": "v7.2.3",
   "PreviewReleaseTag": "v7.3.0-preview.3",
   "ServicingReleaseTag": "v7.0.9",
-  "ReleaseTag": "v7.2.2",
-  "LTSReleaseTag" : ["v7.2.2", "v7.0.9"],
+  "ReleaseTag": "v7.2.3",
+  "LTSReleaseTag" : ["v7.2.3", "v7.0.9"],
```

Comment:
Are we going to release 7.0.10 this time? If so, maybe other that as well?

Reply:
yes; updated.